Builds/CMake/CMakeFuncs.cmake

```diff
@@ -35,6 +35,7 @@ macro(parse_target)
       string(REGEX REPLACE "^\\.?[^\\.]+(.*$)" "\\1" remaining ${remaining})
 
       if (${cur_component} STREQUAL gcc)
+        set(expect_cc true)
```

Comment:
A: It was an edge case setting up Ninja builds under Windows.
https://github.com/ximinez/rippled/blob/4e693d611ef8965114f3a5689e9f9f8b4476187f/CMakeLists.txt#L31-L32
> * Ninja command line builds seem to work under Windows, but only from within
>    the "VS2015 x86 x64 Cross Tools" Command Prompt.
Without this change, if I'm under that command prompt, and run `cmake ../../.. -GNinja`, the `project` command complains that it can't find a compiler - because this block wrote out an empty string.

I'll add a brief comment.